### DISCUSSION TITLE
挂载123云盘出现：签名错误:dykey illegality 1001

### DISCUSSION THREAD

**Discussion by 520yxl** [AlistGo/alist#4850] [Q&A] [answered]
failed get objs: failed to list objs: 签名错误:dykey illegality 1001

**Comment by 520yxl** [comment]
![image](https://github.com/alist-org/alist/assets/29514859/17c7dce8-f2b9-45ae-8965-931f4349f3cb)

**Comment by shaobingtongzhi** [comment]
我也是

**Comment by liyangmoney** [comment]
我也是啊，请问有大神解决了吗？

**Reply by shaobingtongzhi** [reply]
估计得等一个新版本了

**Comment by 1062930629** [comment]
failed get objs: failed to list objs: 签名错误:dykey illegality 1001


我也是

**Comment by yqq2077** [comment]
![image](https://github.com/alist-org/alist/assets/130887008/530255a2-c936-47c6-850d-3984cfab2749)
同问题

**Comment by huolanyun** [comment]
等待更新

**Comment by 520yxl** [comment, marked as answer]
已弃用123云盘了，真的搞死人了
